app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
Indirectly :) Previously path would've defaulted to `/controller` and then the line you did change 209 would've made it `controller/auto_complete_search`